chocolatey/tools/chocolateyInstall.ps1

```diff
@@ -1,55 +1,40 @@
-﻿function Insert-Script([ref]$originalScript, $script) {
-    if(!($originalScript.Value -Contains $script)) { $originalScript.Value += $script }
-}
-
-try {
+﻿try {
     $binRoot = Get-BinRoot
-
-    $oldPromptOverride = "if(Test-Path Function:\Prompt) {Rename-Item Function:\Prompt PrePoshGitPrompt -Force}"
-    $newPromptOverride = "function Prompt() {if(Test-Path Function:\PrePoshGitPrompt){++`$global:poshScope; New-Item function:\script:Write-host -value `"param([object] ```$object, ```$backgroundColor, ```$foregroundColor, [switch] ```$nonewline) `" -Force | Out-Null;`$private:p = PrePoshGitPrompt; if(--`$global:poshScope -eq 0) {Remove-Item function:\Write-Host -Force}}PoshGitPrompt}"
-
     $poshgitPath = join-path $binRoot 'poshgit'
 
     try {
       if (test-path($poshgitPath)) {
-        Write-Host "Attempting to remove existing `'$poshgitPath`' prior to install."
+        Write-Host "Attempting to remove existing `'$poshgitPath`'."
         remove-item $poshgitPath -recurse -force
       }
     } catch {
-      Write-Host 'Could not remove poshgit folder'
+      Write-Host "Could not remove `'$poshgitPath`'"
     }
 
-    $poshGitInstall = if($env:poshGit -ne $null){ $env:poshGit } else {'https://github.com/dahlbyk/posh-git/zipball/master'}
+    $poshGitInstall = if ($env:poshGit ) { $env:poshGit } else { 'https://github.com/dahlbyk/posh-git/zipball/master' }
     Install-ChocolateyZipPackage 'poshgit' $poshGitInstall $poshgitPath
-    $pgitDir = Dir "$poshgitPath\*posh-git*\" | Sort-Object -Property LastWriteTime | Select -Last 1
+    $currentVersionPath = Get-ChildItem "$poshgitPath\*posh-git*\" | Sort-Object -Property LastWriteTime | Select-Object -Last 1

```

Comment:
This is really just defensive programming. A few lines up we try to `remove-item $poshgitPath`, so this folder will almost always have a single subfolder.